tsdb/tombstones/tombstones.go

```diff
@@ -30,7 +32,7 @@ import (
 	"github.com/prometheus/prometheus/tsdb/fileutil"
 )
 
-const tombstoneFilename = "tombstones"
+const Filename = "tombstones"
```

Comment:
This can be confusing to use within the package. Rather keep it `TombstoneFilename`?